### ISSUE TITLE
Request: Customizing or expanding existing parameter

### ISSUE THREAD

**Issue by Mod-lab-stoff** [nikobojs/manuscrape_electron#56]
Could it be possible to add a metadata option for a parameter?
Specifically to a dropdown menu. A new type of data or coding need might arise in the middle of a project. 
And i assume if i delete a parameter and create a new one with the extra option the labeling of old observations are now unreadable.

**Comment by nikobojs** [comment]
Yes, this seems like an important feature and is definitely doable

**Comment by nikobojs** [comment]
related: https://github.com/nikobojs/manuscrape_electron/issues/6

**Comment by nikobojs** [comment]
It is possible now to add/remove multiple choice options on existing projects without affecting existing observations. Is that what you are looking for ? I know editing of parameters are still limited in general but that's a different issue (#6)
